src/styles/blocks/header.scss

```diff
@@ -0,0 +1,68 @@
+.header {
+  justify-content: space-between;
+  background-color: $white;
+  display: flex;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    font-size: 0;
+  }
+}
+
+.nav {
```

Comment:
it should be either a separate file as it is an independent block, or it should be an element of header block. If you already did so, then there is no need to reset them once again. Please, check it for all other classes